docs/getting-started/service-entities-roles-permissions.md

```diff
@@ -209,7 +200,7 @@ Use Code View to view and explore the generated code. You can see the updated co
 
 3. After the build process completes, click the download icon in the _Generate Code_ row to get a .zip file of your app. It is now ready to be deployed anywhere you want. 🚀
```

Comment:
This is not updated, we no longer have an option to download a zip file - only to sync to GitHub and see your code there (or Bitbucket if you're an Enterprise plan user).
@dericksozo can you review this part in the actual product to see how it looks like and fix the doc accordingly?

Reply:
Hey @mulygottlieb @GreenMachine01, I updated the steps in this file to more correctly reflect what is happening in the app. I also split up some of the content at the end into a separate page because its goal is different from the current page. Please let me know if this makes sense.